transaction-status/src/parse_token.rs

```diff
@@ -212,7 +214,10 @@ pub fn parse_token(
         } => {
             check_num_token_accounts(&instruction.accounts, 2)?;
             let owned = match authority_type {
-                AuthorityType::MintTokens | AuthorityType::FreezeAccount => "mint",
+                AuthorityType::MintTokens
+                | AuthorityType::FreezeAccount
+                | AuthorityType::TransferFeeConfig
+                | AuthorityType::WithheldWithdraw => "mint",
```

Comment:
This is a bit of a tricky situation... in token-2022, we can close mints, so `CloseAccount` applies to either a mint or an account.  I'm not seeing an easy way to parse this out.  Our best option is to make a separate `MintCloseAuthority` enum on the program itself.  What do you think?

Reply:
As discussed offline, we added an AuthorityType::CloseMint variant to disambiguate the account being modified by a set_authority instruction